### ISSUE TITLE
loss of accuracy after porting keras trained  model to tflite and using ammnn TFlite parser

### ISSUE THREAD

**Issue by akanxit** [ARM-software/armnn#376]
Hi, 
I trained keras hdf5  cnn model with 5 classes. Then to run inference on arm board, I ported the model to tflite using *tf.lite.TFLiteConverter.from_keras_model(model)* (tensorflow 2.1.0) as mentioned in
https://www.tensorflow.org/api_docs/python/tf/lite/TFLiteConverter.
Then I used the same model to perform inference on a bunch of test images. The classification accuracy drops from 92% on hdf5 model  to 78.96 % using armnn-tflite parser inference.
What could be the possible reason for accuracy drop ? What factors should I look into?
Any help would be appreciated.

**Comment by MikeJKelly** [comment]
Is your model quantized? You can see a significant drop in accuracy on quantized models.

**Comment by Burton2000** [comment]
Do you also experience this accuracy drop if you try running the tflite model with the tflite interpreter (https://www.tensorflow.org/lite/guide/inference#load_and_run_a_model_in_python)?
